WMF Framework/MediaWikiAcceptLanguageMapping.json

```diff
@@ -11,16 +10,6 @@
         },
         "hans": {
             "default": "zh-hans",
-            "cn": "zh-cn",
-            "hk": "zh-hk",
-            "mo": "zh-mo",
-            "my": "zh-my",
-            "sg": "zh-sg",
-            "tw": "zh-tw"
-        },
-        "hant": {
```

Comment:
This mapping is going from OS locale identifiers to Wikipedia variants.

So, hant is still a valid OS locale identifier which we need to handle. It's where we map to the correct variants of Chinese, Traditional, which is still needed.

I've added to the ticket, since we also need to choose which variant of Traditional Chinese we will use if the user has 'Chinese, Traditional' selected. (It seems wrong to choose simplified if the user has clearly preferred traditional.)

(I wish you could put comments in json!)

Reply:
The changes look good.